src/time-picker/time-picker-dialog.jsx

```diff
@@ -81,7 +81,6 @@ let TimePickerDialog = React.createClass({
 
   show() {
     this.refs.dialogWindow.show();
-    this.refs.clock.init();
```

Comment:
This is no longer needed. The component is mounted every time show() is trigger.